app/views/layouts/application.html.erb

```diff
@@ -4,11 +4,16 @@
       breadcrumbs: @breadcrumbs,
       collapse_on_mobile: true
     } %>
-    <main id="content" role="main">
+    <main id="content" role="main" class="govuk-main-wrapper--l">
```

Comment:
Suggestion: when you write PRs like this can you include links to the pages that are impacted? It makes it easier for people to compare changes and review, thanks.

Reply:
Ah yes sorry, I meant to do that but forgot with juggling various PRs. I'll make sure to do it for the others.